src/app/containers/Video/index.jsx

```diff
@@ -60,10 +78,39 @@ const VideoContainer = ({ blocks }) => {
               {JSON.stringify(metadata)}
             </script>
           }
+          <script type="text/javascript">
+            {`
+              function mediaPlayerSetup(container) {
+                  require(['bump-4'], (bump) => {
+                    var mediaPlayer = bump.player(
+                      document.getElementById('mediaPlayer'),
+                      ${JSON.stringify(mediaPlayerSettings)});
+                    mediaPlayer.load();
+                  });
+              }
+            `}
+          </script>
+          <script type="text/javascript">
+            {`
+              function initialiseRequires() {
+                  requiredScripts = {
+                    "bump-4":"//emp.bbci.co.uk/emp/bump-4/bump-4"
```

Comment:
I was requiring jquery here too, based on the SMP embedding docs saying you would need it if you don't have Barlesque on the page. 
_however_ jquery isn't needed for bump 4